docs/review_process.md

```diff
@@ -0,0 +1,143 @@
+Once you're done with your changes, remember to convert from "Draft" to "Normal" pull request and remove WIP keyword from the title.
+
+## Getting your pull request reviewed
+
+Each PR must be reviewed by several reviewers before it will be merged. It cannot be just reviews from random people, we have two categories of reviewers:
+
+### Official reviewers
+
+The list includes only official Conan developers:
+
+- [@memsharded](https://github.com/memsharded)
+- [@lasote](https://github.com/lasote)
+- [@danimtb](https://github.com/danimtb)
+- [@jgsogo](https://github.com/jgsogo)
+- [@czoido](https://github.com/czoido)
+- [@solvingj](https://github.com/solvingj)
+- [@sse4](https://github.com/sse4)
+- [@uilianries](https://github.com/uilianries)
+
+### Community reviewers
+
+The list includes conan-center-index contributors who are very active and proven to be trusted - they frequently submitted pull requests and provide their own useful reviews:
+
+- [@madebr](https://github.com/madebr)
+- [@SpaceIm](https://github.com/SpaceIm)
+- [@ericLemanissier](https://github.com/ericLemanissier)
+- [@prince-chrismc](https://github.com/prince-chrismc)
+- [@Croydon](https://github.com/Croydon)
+- [@intelligide](https://github.com/intelligide)
+- [@theirix](https://github.com/theirix)
+- [@gocarlos](https://github.com/gocarlos)
+
+The list is not constant and will change periodically based on contribution.
+That also means you can be included in this list as well - submit PRs and provide reviews, and in time you may be added as a trusted contributor.
+
+### Rule of 3 reviews
+
+At least 3 reviews are required for an approval, and at least one of them has to be from the official reviewers.
+So, it might be 1 official + 2 community, or 3 official, but it couldn't be just 3 community reviews.
+
+### Reviews from others
+
+All reviews are still valuable and very helpful. Even if you're not listed as an official or community reviewer counted by the bot, your reviews are still welcome, so please do not hesitate to provide them.
+
+### Addressing review comments
+
+Please ensure to address the review comments and respond to them in order to get your PR approved and finally merged.
+It doesn't always mean accepting all the suggestions, but at least providing a response, so people can understand your position.
+
+## Automatic Merges
+
+The bot runs Automatic Merges every 30 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
+PR is selected for the merge only if:
+
+- Author is an added [beta user](https://github.com/conan-io/conan-center-index/issues/4).
+- Author has signed CLA.
+- PR is not a Draft or WIP.
+- PR has a green status (successful build).
+- PR doesn't have merge conflicts with `master` branch.
+- PR has 3 approved reviews (as described above).
```

Comment:
Since this PR is blocked.... might be worth adding that too!

```suggestion
- PR has 3 approved reviews (as described above).
- PR does not have any [official reviewers](#official-reviewers) requesting changes
```